gps/app/Entry.js

```diff
@@ -31,15 +30,16 @@ class Entry extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      initialRouteName: '',
+      onboardingComplete: null,
```

Comment:
this state piece has been used as a flag to determine this, so made the naming explicit